server/example.py

```diff
@@ -23,9 +23,49 @@
 from persona_link.avatar.models import AvatarInput, WebhookResponseData
 
 app = FastAPI()
-
+import requests
+import json
 # add cors policy to allow localhost:3000 and localhost:9000
+def init_avatar():
```

Comment:
You must check if avatar already sxist before issuing a create request as create would create another one with a different slug.